kubernetes/samples/scripts/domain-lifecycle/startCluster.sh

```diff
@@ -0,0 +1,105 @@
+# !/bin/sh
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/../common/utility.sh
+source ${scriptDir}/../common/validate.sh
+source ${scriptDir}/helper.sh
+
+function usage() {
+
+  cat << EOF
+
+  This is a helper script for starting a cluster by patching
+  it's 'spec.serverStartPolicy' field to 'IF_NEEDED'. This change will cause
+  the operator to initiate startup of cluster's WebLogic server pods if the 
+  pods are not already running.
+ 
+  Usage:
+ 
+    $(basename $0) -c mycluster [-n mynamespace] [-d mydomainuid]
+  
+    -c <cluster>        : Cluster name parameter is required.
+
+    -d <domain_uid>     : Default is 'sample-domain1'.
+
+    -n <namespace>      : Default is 'sample-domain1-ns'.
+
+    -m <kubernetes_cli> : Kubernetes command line interface. Default is 'kubectl'.
+
+    -h                  : This help.
+   
+EOF
+exit $1
+}
+
+set -e
```

Comment:
Since all vars seem to be set to some value or are accessed in default expressions, maybe use "set -eu"  instead of "set -e" in order to assert that you don't use an undefined var?

Reply:
Made changes to use "set -eu" instead of "set -e"

Reply:
LGTM